component_generator.thor

```diff
@@ -33,6 +33,10 @@ class ComponentGenerator < Thor::Group
     template("templates/test.tt", "test/components/#{underscore_name}_test.rb")
   end
 
+  def create_integration_test
+    template("templates/integration_test.rb.tt", "test/integration/#{underscore_name}_test.rb") if js_package_name
+  end
```

Comment:
we'd also need to generate a preview like https://github.com/primer/view_components/blob/main/demo/test/components/previews/primer/counter_component_preview.rb